tests/beaker_tests/snmp_notification_receiver/snmp_notification_receiver_provider_defaults.rb

```diff
@@ -53,6 +53,12 @@
 result = 'PASS'
 testheader = 'snmp_notification_receiver Resource :: All Attributes Defaults'
 
+if operating_system == 'ios_xr'
+  interface = 'gigabitethernet0/0/0/1'
```

Comment:
@mikewiebe we are actually changing the interface used during the test. it is a subtle in the code.